collectors/all.h

```diff
@@ -266,65 +266,76 @@
 
 // IP STACK
 
-#define NETDATA_CHART_PRIO_IP_ERRORS                  4100
-#define NETDATA_CHART_PRIO_IP_TCP_CONNABORTS          4210
-#define NETDATA_CHART_PRIO_IP_TCP_SYN_QUEUE           4215
-#define NETDATA_CHART_PRIO_IP_TCP_ACCEPT_QUEUE        4216
-#define NETDATA_CHART_PRIO_IP_TCP_REORDERS            4220
-#define NETDATA_CHART_PRIO_IP_TCP_OFO                 4250
-#define NETDATA_CHART_PRIO_IP_TCP_SYNCOOKIES          4260
-#define NETDATA_CHART_PRIO_IP_TCP_MEM                 4290
-#define NETDATA_CHART_PRIO_IP_BCAST                   4500
-#define NETDATA_CHART_PRIO_IP_BCAST_PACKETS           4510
-#define NETDATA_CHART_PRIO_IP_MCAST                   4600
-#define NETDATA_CHART_PRIO_IP_MCAST_PACKETS           4610
-#define NETDATA_CHART_PRIO_IP_ECN                     4700
+#define NETDATA_CHART_PRIO_IP_TCP_PACKETS             4200
+#define NETDATA_CHART_PRIO_IP_TCP_ERRORS              4210
+#define NETDATA_CHART_PRIO_IP_TCP_ESTABLISHED_CONNS   4220
+#define NETDATA_CHART_PRIO_IP_TCP_OPENS               4220
```

Comment:
Hi Ilya, these 2 share the same priority ?

Reply:
Indeed. But that is minor and can be ignored, not worth changing (=> ci again).